components/Tag.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import { StyleSheet } from 'react-native';
+import { Text } from 'react-native';
+
+export default class Tag extends Component<Props> {
+	constructor() {
```

Comment:
if this class has no methods let's make it a functional component (SFC)
https://reactjs.org/docs/components-and-props.html

Reply:
moreover, I think this is simple enough that it could just be in the same file as `TagList`

Reply:
hmmm, so I don't actually use this component, I just forgot to delete it altogether